### PR TITLE
Efficientipfix

### PR DESCRIPTION
**Pull request by gjackson-eip**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by gjackson-eip** [comment]
Adding template for EfficientIP VHD

File: 201-vm-efficientip-vhd/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    "dnsNameForPublicIP": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+      }
+    }
+  },
+  "variables": {
+    "diagStorageAccountName": "[concat(uniquestring(resourceGroup().id), 'specvm')]",
+    "api-version": "2015-06-15",
```
Comment (bmoore-msft) [inline 201-vm-efficientip-vhd/azuredeploy.json]: remove this and use a literal apiVersion

File: 201-vm-efficientip-vhd/azuredeploy.parameters.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (bmoore-msft) [inline 201-vm-efficientip-vhd/azuredeploy.parameters.json]: format the JSON

File: 201-vm-efficientip-vhd/azuredeploy.parameters.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+		"existingSecurityGroupName": {
+            "value": "GET-PREREQ-existingSecurityGroupName"
+        },
+        "osDiskVhdUri": {
+            "value": "GEN-SPECIALIZED-WINVHD-URI"
+        },
+        "vmName": {
+            "value": "GEN-UNIQUE-8"
+        },
+        "osType": {
```
Comment (bmoore-msft) [inline 201-vm-efficientip-vhd/azuredeploy.parameters.json]: remove these last 3 and just use defaultValues

File: 201-vm-efficientip-vhd/azuredeploy.json (hunk 1 above)
Comment (gjackson-eip) [reply 201-vm-efficientip-vhd/azuredeploy.json]: I am not sure I understand.  This is a API version.  What is wrong with this version?

File: 201-vm-efficientip-vhd/azuredeploy.parameters.json (hunk 2 above)
Comment (gjackson-eip) [reply 201-vm-efficientip-vhd/azuredeploy.parameters.json]: Ok. I am looking at the JSON in notepad and VS.  I don't see anything wrong with the format.  Please explain what you would like me to correct?

File: 201-vm-efficientip-vhd/azuredeploy.parameters.json (hunk 3 above)
Comment (gjackson-eip) [reply 201-vm-efficientip-vhd/azuredeploy.parameters.json]: Understand this one.  Changes has been made

File: 201-vm-efficientip-vhd/azuredeploy.parameters.json (hunk 2 above)
Comment (bmoore-msft) [reply 201-vm-efficientip-vhd/azuredeploy.parameters.json]: in VS Code - do SHIFT+ALT+F

Comment (gjackson-eip) [reply 201-vm-efficientip-vhd/azuredeploy.parameters.json]: Thanks removed the tabs

**Pull request merged**